client/src/main/scala/org/zalando/nakadi/client/handler/SubscriptionHandler.scala

```diff
@@ -84,7 +84,15 @@ class SubscriptionHandlerImpl(val connection: Connection) extends SubscriptionHa
     val result = Source(List(cursor))
       .via(requestFlow)
       .withAttributes(ActorAttributes.supervisionStrategy(decider()))
-      .runForeach(_.throttle(streamParams.rate.get,1.second,streamParams.burstMaxSize.get,ThrottleMode.shaping).runWith(Sink.fromSubscriber(subscriber)))
```

Comment:
Why did you move default values here. I think they were better visible in the StreamParameters class